functions/src/models/slack/messageBlocks.ts

```diff
@@ -246,7 +248,11 @@ export const createFeedbackBlocks = (
         `*Exercise:*\n${exercise} (${sessionMode ? sessionMode : ''} - ${
           sessionType ? sessionType : ''
         })\n\n` +
-        `*Comment:*\n${comment}`,
+        `*Comment:*\n${comment}\n\n` +
+        `*Model:* ${params.model ?? ''}\n\n` +
+        `*OS:* ${params.os ?? ''}\n\n` +
+        `*Native Version:* ${params.nativeVersion ?? ''}\n\n` +
+        `*Bundle Version:* ${params.bundleVersion ?? ''}`,
```

Comment:
This will probably swell the message vertically a lot. But let's keep an eye on it and maybe put it in two columns if needed. There's a specific slack message block for that. Like with the old app
<img width="624" alt="image" src="https://github.com/29ki/29k/assets/474066/058c9c66-7a94-4223-bb06-5d70421a0018">